driver/SDL/modSDL.c

```diff
@@ -25,11 +29,22 @@ STATIC int tick_thread(void * data)
 
     return 0;
 }
+#else
+STATIC void mp_lv_main_loop(void)
+{
+        mp_sched_schedule((mp_obj_t)&mp_lv_task_handler_obj, mp_const_none);
+        lv_tick_inc(2);
+}
+#endif
 
 STATIC mp_obj_t mp_init_SDL()
 {
     monitor_init();
+#ifdef __EMSCRIPTEN__
+    emscripten_set_main_loop(mp_lv_main_loop, 500, 0);
```

Comment:
500 FPS, and `lv_tick_inc(2)` must be related right?
What do you think about defining some value and deriving these values from it?

Reply:
Good idea. I just pushed a commit which defines `LV_TICK_RATE`.